content/en/docs/concepts/security/multi-tenancy.md

```diff
@@ -0,0 +1,235 @@
+
+The remainder of this page focuses on isolation techniques used for shared Kubernetes clusters. However, even if you are considering dedicated clusters, it may be valuable to review these recommendations, as it will give you the flexibility to shift to shared clusters in the future if your needs or capabilities change.
+
+
+## Control Plane Isolation
+
+Control plane isolation ensures that different tenants cannot access or affect each others' Kubernetes API resources. 
+
+### Namespaces
+
+In Kubernetes, a Namespace provides a mechanism for isolating groups of API resources within a single cluster. This isolation has two key dimensions:
+
+1. Object names within a namespace can overlap with names in other namespaces, similarly to files in folders. This allows tenants to name their resources without having to consider what other tenants are doing.
+
+2. Many Kubernetes security policies are scoped to namespaces. For example, RBAC and Network Policies are scoped to namespaces (by default), while objects such as Secrets and Service Accounts may only be accessed from their own namespaces.
+
+In a multi-tenant environment, a Namespace helps segment a tenant's workload into a logical and distinct management unit. In fact, a common practice is to isolate every workload in its own namespace, even if multiple workloads are operated by the same tenant. This ensures that each workload has its own identity and can be configured with an appropriate security policy.
+
+The namespace isolation model requires configuration of several other Kubernetes resources, networking plugins, and adherence to security best practices to properly isolate tenant workloads. These considerations are discussed below.
+
+### Access Controls
+
+The most important type of isolation for the control plane is authorization. If teams, or their workloads, can access or modify each others' API resources, they can change or disable all other types of policies, negating any protection those policies may offer. As a result, it is critical to ensure that each tenant has the appropriate access to only the namespaces they need, and no more. This is known as the "Principle of Least Privilege."
+
+Role-based access control (RBAC) is the recommended method to enforce authorization in the Kubernetes control plane, for both human users and workloads (service accounts). [Roles](/docs/reference/access-authn-authz/rbac/#role-and-clusterrole) and [role bindings](/docs/reference/access-authn-authz/rbac/#rolebinding-and-clusterrolebinding) are Kubernetes objects that are used at a namespace level to enforce access control in your application; similar objects exist for authorizing access to cluster-level objects, though these are less useful for multi-tenant clusters.

```

Comment:
Who is recommending RBAC? I assume the Kubernetes project; however, we should make that clear.
```suggestion
Role-based access control (RBAC) is the recommended method to enforce authorization in the Kubernetes
API, for both human users and workloads ({{< glossary_tooltip text="service account" term_id="service-account" >}}).
[Roles](/docs/reference/access-authn-authz/rbac/#role-and-clusterrole) and [RoleBindings](/docs/reference/access-authn-authz/rbac/#rolebinding-and-clusterrolebinding) are Kubernetes objects that are
used at a namespace level to enforce access control in your application;
similar objects exist for authorizing access to cluster-level objects, though these are less useful for multi-tenant clusters.
```

I'd like to check with SIG Auth whether we actively recommend RBAC or whether we are neutral (perhaps we don't take a view on the merits of in-tree authz vs. extension authz in the multitenant case).